string id
float64 pos_x
float64 pos_y
float64 vel_x
float64 vel_y